### DISCUSSION TITLE
Share Langium Roadmap

### DISCUSSION THREAD

**Discussion by theogiraudet** [eclipse-langium/langium#563] [Ideas]
Hi!
I think is the best category for a question like this.
There is a page or something like this where we can find the Langium roadmap? I want to present Langium to the team in which I am and finish the presentation on "Langium in the future".
If you want to keep it private, no problem, I just ask! :)

**Comment by msujew** [comment]
Hi @theogiraudet,

We haven't really released a roadmap yet, but we're looking into it. We're currently in the process of making Langium more open for "outsiders" and are planning to hold short, weekly dev-meetings on **Wednesdays 14:00 UTC** (4PM Berlin time), starting next week. More info on that shortly in the GitHub wiki (not yet activated).

Anyway, for your presentation, I can give you a sneak-peek into our current roadmap:
* We're planning a 1.0 release likely at the end of this year. We're seeing quite a good amount of maturity in the project already, as we're using Langium extensively in our (closed source) customer projects
* We're putting most of our work in getting grammar related features fixed (mostly type based validation, generation and imports), see the [current milestone](https://github.com/langium/langium/milestone/4). There's a lot of work to be done here
* More out-of-the-box support for LSP services (type definitions, code lenses, signature help, etc.)
* More utilities for handling code generation for downstream users
* Refactoring some existing services (mostly code completion)

**Reply by theogiraudet** [reply]
Thank you!
